### PR TITLE
MS-1164 task: update confirmation heading

### PR DESCRIPTION
**Pull request by jjrobothamho**
- Toggle confirmation heading based on user journey.

### CODE REVIEW

File: apps/nrm/behaviours/confirmation.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const superlocals = super.locals(req, res);
+    const data = Object.assign({}, formatHeading(req));
+    const locals = Object.assign({}, superlocals, data);
+
+    return locals;
+  }
+};
```
Comment (sulthan-ahmed) [inline apps/nrm/behaviours/confirmation.js]: I’d probably rewrite this code

Can this all be simplified like this
https://github.com/UKHomeOffice/modern-slavery/blob/master/apps/verify/behaviours/confirm-email-passer.js

File: test/unit/nrm/behaviours/confirmation.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    });
+    afterEach(() => {
+      Base.prototype.process.restore();
+    });
+
+    it('calls the parent method after updating the page locals', async() => {
+      req.sessionModel.get.withArgs('pv-want-to-submit-nrm').returns('no');
+
+      await instance.process(req, res, next);
+      expect(Base.prototype.process).to.have.been.called;
```
Comment (sulthan-ahmed) [inline test/unit/nrm/behaviours/confirmation.test.js]: I’m not sure about this test, maybe you want to have a test like this
https://github.com/UKHomeOffice/modern-slavery/blob/master/test/unit/verify/behaviours/confirm-email-passer.test.js#L42

**Comment by sulthan-ahmed** [comment]
Test failing

File: test/unit/nrm/behaviours/confirmation.test.js (hunk 2 above)
Comment (jjrobothamho) [reply test/unit/nrm/behaviours/confirmation.test.js]: Cheers I'll update the codebase

**Pull request merged**